### PR TITLE
Add the electron object detection sample

### PR DESCRIPTION
**Pull request by lionkunonly**

### CODE REVIEW

File: node/examples/electron/object_detection/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# An Electron.js example of Object_Detection reusing webnn-native
```
Comment (fujunwei) [inline node/examples/electron/object_detection/README.md:1]: Object_Detection => Object Detection

Comment (fujunwei) [reply node/examples/electron/object_detection/README.md:1]: reusing => using

File: node/examples/electron/object_detection/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+You can also run multiple times with "numRuns" to get the median inference time, for example:
+```bash
+npm start numRuns=100
+```
+
+### Screenshot
+
+![screenshot](screenshot.png)
```
Comment (fujunwei) [inline node/examples/electron/object_detection/README.md:23]: Do you upload the screenshot?

File: node/examples/electron/object_detection/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "object_detection",
+  "version": "0.0.1",
+  "description": "An Electon.js example of object_detection reusing webnn-native",
```
Comment (fujunwei) [inline node/examples/electron/object_detection/package.json:4]: object_detection reusing => Object Detection using

File: node/examples/electron/object_detection/README.md (hunk 1 above)
Comment (lionkunonly) [reply node/examples/electron/object_detection/README.md:1]: Done!

File: node/examples/electron/object_detection/README.md (hunk 2 above)
Comment (lionkunonly) [reply node/examples/electron/object_detection/README.md:23]: Sorry, I forget to add such a img. I have uploaded it now.

File: node/examples/electron/object_detection/package.json (hunk 3 above)
Comment (lionkunonly) [reply node/examples/electron/object_detection/package.json]: Done

Comment (fujunwei) [reply node/examples/electron/object_detection/package.json]: Do you miss this modification?

**Review by fujunwei — APPROVED** [review]
LGTM, could you please squash to one commit?

**Comment by lionkunonly** [comment]
> LGTM, could you please squash to one commit?

Done. But this sample needs the [PR](https://github.com/webmachinelearning/webnn-samples/pull/62)  in `webnn-samples` .

**Pull request merged**